k8s.gcr.io/images/k8s-staging-cloud-provider-gcp/images.yaml

```diff
@@ -1,14 +1,16 @@
 - name: gcp-compute-persistent-disk-csi-driver
   dmap:
+    "sha256:02ffdc33eaa1c53342d9962ac216a3482bb42aaae12871371bc9594ce94577a2": ["v1.3.2-rc1"]
```

Comment:
Do we promote release candidates? I thought we just tested them out of staging, then promoted the actual release only.

Reply:
we can promote, I can drop that as well

Reply:
Yeah, let's just promote the final candiate and not the RCs to keep the image registry a little cleaner.

Reply:
sgmt!

Reply:
done @mattcary thanks for the review! PTAL